src/python.cc

```diff
@@ -1308,7 +1308,7 @@ ModelInstanceState::SetupStubProcess()
 {
   std::string kind = TRITONSERVER_InstanceGroupKindString(kind_);
   std::string shm_region_name =
-      std::string("/") + Name() + "_" + kind + "_" + std::to_string(device_id_);
+      std::string("/") + Name() + "_" + std::to_string(Model()->Version()) + "_" + kind + "_" + std::to_string(device_id_);
```

Comment:
Could you pass `shm_region_name` to `StartStubProcess` at the end of the function so that the region name is not built multiple times?